components/usage/pkg/db/types.go

```diff
@@ -92,12 +92,35 @@ func (n VarcharTime) String() string {
 	return ""
 }
 
-func (u VarcharTime) MarshalJSON() ([]byte, error) {
-	if !u.IsSet() {
-		return []byte(""), nil
+var null = "null"
+
+func (n VarcharTime) MarshalJSON() ([]byte, error) {
+	if !n.IsSet() {
+		return []byte(null), nil
+	}
+
+	return n.Time().UTC().MarshalJSON()
+}
+
+func (n *VarcharTime) UnmarshalJSON(data []byte) error {
```

Comment:
Was necessary for the report download to serialise correctly. We'd convert it into a string, but on unmarshal we'd try to put it into the VarcharTime struct which would fail